app/src/stores/fields.ts

```diff
@@ -89,6 +89,10 @@ export const useFieldsStore = defineStore({
 			}
 
 			if (field.meta?.conditions) {
+				field.meta.conditions_raw = field.meta.conditions.map((condition) => ({
+					...condition,
+					rule: condition.rule,
+				}));
```

Comment:
Instead of having two copies of the same object, I think it makes more sense to only parse the variables out at the time we're actually using the rule, and keep the one in the original store as is 👍🏻

Reply:
@rijkvanzanten Yes definitely! I did not wish to cause any unwanted side effects. 
Looking forward to more tests such as in #8245~ 😁